exercises/go-1-a/.meta/example_imperative.go

```diff
@@ -0,0 +1,34 @@
+package go1a
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func concat(description string, str string) string {
+	if description != "" {
+		return description + " or " + str
+	}
+
+	return description + str
+}
+
+func Describe(amount int) string {
+	var description string
+
+	if amount%144 == 0 {
+		description += concat(description, fmt.Sprintf("%d gross", amount/144))
```

Comment:
should that be just `=` instead of `+=` ?

Reply:
Right...
```suggestion
		description = concat(description, fmt.Sprintf("%d gross", amount/144))
```

Reply:
It should be, although it didn't matter in this case as `description` was empty at that point. Fixed it.